### PR TITLE
Fix server side support

### PR DESCRIPTION
**Pull request by Kikobeats**
We should `componentDidMount` instead of `componentWillMount` to render a component from server side that need to fetch data content.

### CODE REVIEW

File: packages/react-microlink/src/index.js [hunk 1]
```diff
@@ -5,8 +5,8 @@ import {CardWrap, CardMedia, CardContent, CardEmptyState} from './components/Car
 import {getUrlPath, someProp, createApiUrl} from './utils'
 
 class Microlink extends Component {
-  constructor (props) {
-    super(props)
```
Comment (Kikobeats) [inline packages/react-microlink/src/index.js]: pass `props` is a convention for inherit components

Comment (breadadams) [reply packages/react-microlink/src/index.js]: Hmm, you're probably right, but what's the usecase? I thought it was only necessary to pass `props` if they were used in the constructor.

Comment (Kikobeats) [reply packages/react-microlink/src/index.js]: For example, when you create a component based on `MicrolinkCard`:

```jsx
Class MyCustomComponent extend MicrolinkCard {
 constructor(props) {
   super(props) ← MicrolinkCard constructor
 }
}
```

Comment (Kikobeats) [reply packages/react-microlink/src/index.js]: yes you are right, just was a convention, no problem avoiding passing it 👍

**Pull request merged**